include/dxc/DxilContainer/DxilContainerReader.h

```diff
@@ -18,7 +18,7 @@
 
 namespace hlsl {
 
-#define DXIL_CONTAINER_BLOB_NOT_FOUND -1
+const uint32_t DXIL_CONTAINER_BLOB_NOT_FOUND = -1;
```

Comment:
Doesn't this generate a warning with MSVC?  Or have we disabled that warning (eek)?

```
> type t.cpp ; cl /c /Wall t.cpp
#include <stdint.h>

const uint32_t foo = -1;
Microsoft (R) C/C++ Optimizing Compiler Version 19.37.32824 for x64
Copyright (C) Microsoft Corporation.  All rights reserved.

t.cpp
t.cpp(3): warning C4245: 'initializing': conversion from 'int' to 'uint32_t', signed/unsigned mismatch
t.cpp(3): warning C5264: 'foo': 'const' variable is not used
```